module/tools/SystemConfiguration/data/system/nugus4/etc/systemd/network/10-bond0.netdev

```diff
@@ -0,0 +1,7 @@
+[NetDev]
```

Comment:
I'm thinking for all these (and other identical files) that are the same you should just leave them as default, that way if we need to update them at some point it's not too much work

Reply:
bond, probably. There others (eno1 and wlan0) may be different on different robots which is why I made them device files